src/main/java/use_cases/add_edit_category_use_case/CategoryOB.java

```diff
@@ -0,0 +1,26 @@
+package use_cases.add_edit_category_use_case;
+
+/**
+ * An interface Category_Output_Boundary containing methods implemented in Category_Presenter.
+ */
+public interface CategoryOB {
+    /**
+     * Returns same Category_Output_Data with given Category_Output_Data when category is successfully added.
```

Comment:
I'm having some trouble understanding this JavaDoc. Maybe you could rewrite to make it a bit clearer?